app/keepers/keepers.go

```diff
@@ -0,0 +1,82 @@
+package keepers
+
+import (
+	authkeeper "github.com/cosmos/cosmos-sdk/x/auth/keeper"
+	authzkeeper "github.com/cosmos/cosmos-sdk/x/authz/keeper"
+	bankkeeper "github.com/cosmos/cosmos-sdk/x/bank/keeper"
+	capabilitykeeper "github.com/cosmos/cosmos-sdk/x/capability/keeper"
+	crisiskeeper "github.com/cosmos/cosmos-sdk/x/crisis/keeper"
+	distrkeeper "github.com/cosmos/cosmos-sdk/x/distribution/keeper"
+	evidencekeeper "github.com/cosmos/cosmos-sdk/x/evidence/keeper"
+	feegrantkeeper "github.com/cosmos/cosmos-sdk/x/feegrant/keeper"
+	govkeeper "github.com/cosmos/cosmos-sdk/x/gov/keeper"
+	mintkeeper "github.com/cosmos/cosmos-sdk/x/mint/keeper"
+	paramskeeper "github.com/cosmos/cosmos-sdk/x/params/keeper"
+	slashingkeeper "github.com/cosmos/cosmos-sdk/x/slashing/keeper"
+	stakingkeeper "github.com/cosmos/cosmos-sdk/x/staking/keeper"
+	upgradekeeper "github.com/cosmos/cosmos-sdk/x/upgrade/keeper"
+	packetforwardkeeper "github.com/cosmos/ibc-apps/middleware/packet-forward-middleware/v6/packetforward/keeper"
+	ibctransferkeeper "github.com/cosmos/ibc-go/v6/modules/apps/transfer/keeper"
+	ibckeeper "github.com/cosmos/ibc-go/v6/modules/core/keeper"
+	evmkeeper "github.com/evmos/ethermint/x/evm/keeper"
+	feemarketkeeper "github.com/evmos/ethermint/x/feemarket/keeper"
+	epochskeeper "github.com/osmosis-labs/osmosis/v15/x/epochs/keeper"
+	gammkeeper "github.com/osmosis-labs/osmosis/v15/x/gamm/keeper"
+	incentiveskeeper "github.com/osmosis-labs/osmosis/v15/x/incentives/keeper"
+	lockupkeeper "github.com/osmosis-labs/osmosis/v15/x/lockup/keeper"
+	poolmanagerkeeper "github.com/osmosis-labs/osmosis/v15/x/poolmanager/keeper"
+	txfeeskeeper "github.com/osmosis-labs/osmosis/v15/x/txfees/keeper"
+
+	delayedackkeeper "github.com/dymensionxyz/dymension/v3/x/delayedack/keeper"
+	denommetadatamodulekeeper "github.com/dymensionxyz/dymension/v3/x/denommetadata/keeper"
+	eibckeeper "github.com/dymensionxyz/dymension/v3/x/eibc/keeper"
+	rollappmodulekeeper "github.com/dymensionxyz/dymension/v3/x/rollapp/keeper"
+	sequencermodulekeeper "github.com/dymensionxyz/dymension/v3/x/sequencer/keeper"
+	streamermodulekeeper "github.com/dymensionxyz/dymension/v3/x/streamer/keeper"
+)
+
+type AppKeepers struct {
+	// keepers
+	AccountKeeper                 authkeeper.AccountKeeper
```

Comment:
I don't think this warrants a package, just keep it in app.go in another struct

also if you call a package keepers you dont need to call the type keepers.AppKeepers, just keepers.App is fine